### DISCUSSION TITLE
exporting .asreview project file, and re-importing it as excel file in new project - good idea?

### DISCUSSION THREAD

**Discussion by Morle2104** [asreview/asreview#1875] [Q&A]
Dear all, 

One of the screeners has made a mistake in the screening and has mistakenly labelled a key paper as 'irrelevant'. However, they have already screened around 2.600 papers, and we don't know where exactly the key paper is in the 'History irrelevant' tab in ASReview. Thus, we cannot change the label within the ASReview environment. 

Our idea is to export the project as a csv file, change the label of the key paper in the csv file, and create a new project in ASReview with this csv file using the same classifier and feature extrator as before. 
From my understanding, ASReview will just use all existing labelled records as prior knowledge. However, we are also losing information on labels that were switched within ASReview before, which is relevant for reproducibility. Thus, our idea is to make a backup of the current .asreview file and save this.

My question is whether any other relevant information is changed or lost through this process of _old .asreview file -> csv file --> new .asreview project._ ? 

Thank you in advance and kind regards!

**Comment by Rensvandeschoot** [comment]
This paper explains in detail what is stored and what is not: https://doi.org/10.3390/app14093842.

Would it be possible to search for this paper in the history panel by using the search functionality of your web browser (e.g., `ctr+f`)?
